airflow/providers/databricks/hooks/databricks.py

```diff
@@ -419,6 +427,46 @@ def get_run_state(self, run_id: str) -> RunState:
         state_message = state['state_message']
         return RunState(life_cycle_state, result_state, state_message)
 
+    def get_run_state_str(self, run_id: str) -> str:
+        """
+        Return the string representation of RunState.
+
+        :param run_id: id of the run
+        :return: string describing run state
+        """
+        state = self.get_run_state(run_id)
+        run_state_str = (
+            f"State: {state.life_cycle_state}. Result: {state.result_state}. {state.state_message}"
+        )
```

Comment:
How about putting this in `RunState`’s `__str__` and just call `str(state)` here?

Reply:
That’s primarily because I’m not sure if it may break something inside airflow. I’m not expert in it (yet)